lib/tarpon/entity/offerings.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'tarpon/entity/offering'
+
+module Tarpon
+  module Entity
+    class Offerings
```

Comment:
Should we make this an enumerable? Gives more flexibility to work with collections and keep consistency as we also do that in the entitlement list [here](https://github.com/fishbrain/tarpon/blob/master/lib/tarpon/entity/entitlement_list.rb#L6).